dev/archery/setup.py

```diff
@@ -36,6 +36,7 @@
                  'setuptools_scm'],
     'crossbow-upload': ['github3.py', jinja_req, 'ruamel.yaml',
                         'setuptools_scm'],
+    'numpydoc': ['numpydoc==1.1.0']
```

Comment:
Is there a particular reason for pinning to this exact version?

Reply:
This it the first version which provides the class and function we use from numpydoc which seem a bit like internal-ish at the moment.